cmake/compile_flags_and_defs.cmake

```diff
@@ -100,12 +100,17 @@ list(APPEND MSVC_LINKER_FLAGS
   /DEBUG                                # Generate debug information (even for release builds for our stackwalker)
   /MANIFEST:NO                          # No default manifest
   /SAFESEH:NO                           # Do Not Create a table of safe exception handlers (binary incompatible)
+
+  )
+list(APPEND MSVC_LINKER_FLAGS_DEBUG
+  /NODEFAULTLIB:MSVCRTD                 # This fixes linking for 32-bit
```

Comment:
this shouldn't fix linking for 32 bit.
Iirc it was a failure in our enet-package to miss setting MT correctly. is this still a thing? if no we could revert this line